package.json

```diff
@@ -75,6 +75,7 @@
     "typescript": "^3.9.5"
   },
   "dependencies": {
+    "@graphql-typed-document-node/core": "3.1.0",
```

Comment:
Is this for TS annotation only? In that case I think it is better to be in `devDependencies`

Reply:
`TypedDocumentNode` is used in the exported types and functions such as `compileQuery` and `CompiledQuery`. If we move this to `devDependencies`, those types and function signatures will break.